ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+
+
+## Possible Solutions
+
```

Comment:
- [ ] Write "(Please write ideas or candidates of solutions for the problem if you have.)"